client/crypto/decrypt.go

```diff
@@ -0,0 +1,149 @@
+package crypto
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	datadealkeeper "github.com/medibloc/panacea-core/v2/x/datadeal/keeper"
+	oraclekeeper "github.com/medibloc/panacea-core/v2/x/oracle/keeper"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/libs/cli"
+)
+
+const (
+	FlagCipherText     = "cipher-text"
+	FlagCipherTextPath = "path"
+)
+
+func Command(defaultNodeHome string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "decrypt-data [file-path] [key-name] [dealID]",
+		Short: "Decrypt and output the data file encrypted with shared key which consists of oracle public key and buyer private key.",
+		Long: `This command can decrypt encrypted data with shared key which consists of oracle public key and buyer private key.
```

Comment:
It seems that this command is for decrypting data which is encrypted for a buyer (aka. delivered data). If so, please rename the PR title to `feat: implement decrypting delivered data` or so.

Reply:
Yes, I think that the PR title can be confused for reviewers. I think the `feat: implement decrypting delivered data` title seems to be good. I'll change it.